src/test/java/com/github/susom/database/test/CommonTest.java

```diff
@@ -98,6 +89,28 @@ public void closeJdbc() throws Exception {
     }
   }
 
+  @Test
+  public void tableExists() {
+    // Verify dbtest table does not exist
+    String lowercaseTable = TEST_TABLE_NAME.toLowerCase();
+    testTableLookup(lowercaseTable);
+    db.dropTableQuietly(lowercaseTable);
+
+    // Let's try creating a table with an upper case name and verify it works
+    String uppercaseTable = TEST_TABLE_NAME.toUpperCase();
+    testTableLookup( uppercaseTable );
+    db.dropTableQuietly(uppercaseTable);
+  }
```

Comment:
This test doesn't cover the double quoted table name behavior you implemented

Reply:
I did not include a test for camel case because I can't create a table in mixed case -> I can't unit test it without further changes to the database code.  I tried it.  The AddTable API blew up with it.  I did manually create a camelcase  table in Postgres and tested it with code I did not check in because I can't create it and it does work.

I thought it was better to leave a backdoor to let people do this if they want/need to, but if you prefer, I will remove that feature. Please advise.

Reply:
I've already registered my opinion (mixed case tables should be out of scope for the library), but I'll defer to Garrick on that.  If we keep the feature, I think we could test `dbTableName` independently in `DatabaseTest`.

Reply:
Good suggestion, Joe.  I will add a test of that there if Garrick wants to keep the Camelcase feature.  Garrick, can you weigh in on whether you want to keep that feature or prefer me to remove it?